lib/index.js

```diff
@@ -285,23 +285,34 @@ TDClient.prototype = {
      *  Submit Hive type job
      *  @param {string} db - The name of database
      *  @param {string} query - The Hive query which run on given database
+     *  @param {object} opts - Optoins supported member are `result`,
```

Comment:
Typo `Optoins` -> `Options`.
I think `Supported options are ~` might be better.